Chapter-2-modules-separation/Src/Common/Fitnet.Common.Api/Fitnet.Common.Api.csproj

```diff
@@ -16,8 +16,9 @@
     </ItemGroup>
 
     <ItemGroup>
-        <PackageReference Include="FluentValidation" Version="11.7.1" />
-        <PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="11.7.1" />
+        <PackageReference Include="FluentValidation" Version="11.9.1" />
+        <PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="11.9.1" />
+        <PackageReference Update="SonarAnalyzer.CSharp" Version="9.25.0.90414" />
```

Comment:
It is defined on the level of Directory.Build.props, please remove it from here

Reply:
Done